newIDE/app/src/EventsSheet/index.js

```diff
@@ -306,6 +306,7 @@ export default class EventsSheet extends React.Component<Props, State> {
   addNewEvent = (type: string, context: ?EventContext): Array<gdBaseEvent> => {
     const { project } = this.props;
     const hasEventsSelected = hasEventSelected(this.state.selection);
+    let insertTopOfSelection = 0;
```

Comment:
What do you think of making this a boolean? and then subtract 0 or 1 according to the boolean?

Reply:
Yes it's better 👍